server/routes/index.ts

```diff
@@ -48,6 +49,8 @@ export default function routes(services: Services): Router {
     services.courtRegisterService,
   )
 
+  const sentencingRoutes = new SentencingRoutes(services.courtAppearanceService)
```

Comment:
Have created a separate sentencing route class - intention being as we split out the journey further we put sentencing routes in here